### PR TITLE
AutoML NLP card

### PR DESCRIPTION
**Pull request by yiyoulin**

### CODE REVIEW

File: assets/models/system/automl-ner/spec.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: AutoML-Named-Entity-Recognition
```
Comment (bhimar) [inline assets/models/system/automl-ner/spec.yaml:2]: nit: maybe AutoML-Text-Named-Entity-Recognition is better, but no strong preference

File: assets/models/system/automl-ner/description.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+For more information, see [Model sweeping and hyperparameter tuning](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=python#model-sweeping-and-hyperparameter-tuning-preview).
+
+### Code samples
+
+
+Task|Dataset|Python sample (Notebook)|CLI with YAML
+|---|--|--|--|
+Named Entity Recognition|[CoNLL-2003](https://www.kaggle.com/datasets/alaakhaled/conll003-englishversion?select=valid.txt)|<a href="https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/automl-standalone-jobs/automl-nlp-text-named-entity-recognition-task-distributed-sweeping/automl-nlp-text-ner-task-distributed-with-sweeping.ipynb" target="_blank">automl-nlp-text-ner-task-distributed-with-sweeping.ipynb</a>|<a href="https://github.com/Azure/azureml-examples/blob/main/cli/jobs/automl-standalone-jobs/cli-automl-text-ner-conll-distributed-sweeping/cli-automl-text-ner-conll2003-distributed-sweeping.yml" target="_blank">/automl-nlp-text-ner-task-distributed-with-sweeping.ipynb</a>
```
Comment (bhimar) [inline assets/models/system/automl-ner/description.md:56]: the link text for cli example is sdk not cli

File: assets/models/system/automl-ner/description.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+three O
+championship O
+rings O
+```
+
+Please see [documentation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=python#named-entity-recognition-ner) for data requirements.
+
+#### Language Setting
+
+Currently, language selection defaults to English. But Automated ML supports 104 languages leveraging language specific and multilingual pre-trained text DNN models. Please see [langugage setting](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=python#language-settings) for documentation.
```
Comment (bhimar) [inline assets/models/system/automl-ner/description.md:43]: typo: langugage -> language

File: assets/models/system/automl-text-classification/description.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+### Documentation
+
+#### Prepare Data
+To create NLP models, it is necessary to provide labeled text data as input for model training. For text classification, the dataset can contain several text columns and exactly one label column. 
+
+Please see [documentation](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=python#preparing-data) for data preparation requirements.
+
+#### Language Setting
+
+Currently, language selection defaults to English. But Automated ML supports 104 languages leveraging language specific and multilingual pre-trained text DNN models. Please see [langugage setting](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=python#language-settings) for documentation.
```
Comment (bhimar) [inline assets/models/system/automl-text-classification/description.md:23]: same typo for "language"

File: assets/models/system/automl-ner/description.md (hunk 2 above)
Comment (yiyoulin) [reply assets/models/system/automl-ner/description.md:56]: thank you for catching that! you have good eyes!

File: assets/models/system/automl-text-classification/description.md (hunk 4 above)
Comment (yiyoulin) [reply assets/models/system/automl-text-classification/description.md:23]: corrected

File: assets/models/system/automl-ner/description.md (hunk 3 above)
Comment (sharma-riti) [inline assets/models/system/automl-ner/description.md]:
```suggestion
Currently, language selection defaults to English. But Automated ML supports 104 languages leveraging language specific and multilingual pre-trained text DNN models. Please see [langugage setting](https://learn.microsoft.com/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=python#language-settings) for documentation.
```

File: assets/models/system/automl-ner/spec.yaml (hunk 1 above)
Comment (yiyoulin) [reply assets/models/system/automl-ner/spec.yaml:2]: Hi @bhimar , I might have to dismiss this comment since I feel it's more natural to say "ner", rather than "text-ner". But I do see your point that you want to have unified name format for NLP tasks.

File: assets/models/system/automl-text-classification/description.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
+Automated Machine Learning, or AutoML, is a process that automates the repetitive and time-consuming tasks involved in developing machine learning models. This helps data scientists, analysts, and developers to create models more efficiently and with higher quality, resulting in increased productivity and scalability.
+
+AutoML Text Classification enables you to classify or categorize texts into predefined groups. Your dataset should be a labeled set of texts with their relevant tags that categorize each piece of text into a predefined group.
+
+With this functionality, you can:
+* Directly use datasets coming from [Azure Machine Learning data labeling](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-text-labeling-projects?view=azureml-api-2)
```
Comment (sharma-riti) [inline assets/models/system/automl-text-classification/description.md:6]: same here... not 100% sure if we support labelling with NLP? @raduk do you know?

File: assets/models/system/automl-text-classification/description.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+* Utilize labeled data to create NLP models without any training code.
+* Enhance model performance by selecting the appropriate algorithm and fine-tuning the hyperparameters selecting the appropriate algorithm from a [large selection of models](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=cli#supported-model-algorithms) or let AutoML find the best model for you.
+* Either download or deploy the resulting model as a endpoint in Azure Machine Learning.
+* Scale the operationalization process with the help of Azure Machine Learning's MLOps and ML Pipelines capabilities.
+
+See [How to train nlp models](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-auto-train-nlp-models?view=azureml-api-2&tabs=cli) for more information.
```
Comment (sharma-riti) [inline assets/models/system/automl-text-classification/description.md:12]: please remove en-us from all links... it sometimes won't open in other regions otherwise...

File: assets/models/system/automl-text-classification/spec.yaml [hunk 7]
```diff
@@ -0,0 +1,51 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: AutoML-Text-Classification
+path: ./
+properties:
+  finetune-min-sku-spec: 4|1|28|176
+  finetune-recommended-sku: Standard_NC12s_v3
+  finetuning-tasks: text-classification
```
Comment (sharma-riti) [inline assets/models/system/automl-text-classification/spec.yaml:7]: can you also please fix this for image segmentation task?

File: assets/models/system/automl-text-classification/description.md (hunk 6 above)
Comment (yiyoulin) [reply assets/models/system/automl-text-classification/description.md:12]: thanks for letting me know!

File: assets/models/system/automl-text-classification/spec.yaml (hunk 7 above)
Comment (yiyoulin) [reply assets/models/system/automl-text-classification/spec.yaml:7]: fixed :)

File: assets/models/system/automl-text-classification/model.yaml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: finetuning-image-models
+  container_path: AutoML-mlflow/AutoML-Image-Classification/1700509519/mlflow_model_folder
```
Comment (sharma-riti) [inline assets/models/system/automl-text-classification/model.yaml:3]:
```suggestion
  container_path: AutoML-mlflow/AutoML-Text-Classification/1701458187/mlflow_model_folder
```

File: assets/models/system/automl-text-classification/model.yaml [hunk 9]
```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: finetuning-image-models
```
Comment (sharma-riti) [inline assets/models/system/automl-text-classification/model.yaml:2]:
```suggestion
  container_name: finetuning-models
```

File: assets/models/system/automl-ner/model.yaml [hunk 10]
```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: finetuning-image-models
+  container_path: AutoML-mlflow/AutoML-Image-Classification/1700509519/mlflow_model_folder
```
Comment (sharma-riti) [inline assets/models/system/automl-ner/model.yaml:3]:
```suggestion
  container_path: AutoML-mlflow/AutoML-Text-Ner/1701459109/mlflow_model_folder
```

File: assets/models/system/automl-ner/model.yaml [hunk 11]
```diff
@@ -0,0 +1,8 @@
+path:
+  container_name: finetuning-image-models
```
Comment (sharma-riti) [inline assets/models/system/automl-ner/model.yaml:2]:
```suggestion
  container_name: finetuning-models
```

File: assets/models/system/automl_instance_segmentation/spec.yaml [hunk 12]
```diff
@@ -5,7 +5,7 @@ properties:
   datasets: COCO
   finetune-min-sku-spec: 4|1|28|176
   finetune-recommended-sku: Standard_NC6s_v3
-  finetuning-tasks: image-object-detection
+  finetuning-tasks: image-instance-segmentation
```
Comment (sharma-riti) [inline assets/models/system/automl_instance_segmentation/spec.yaml:8]:
```suggestion
  finetuning-tasks: image-segmentation
```

Comment (sharma-riti) [reply assets/models/system/automl_instance_segmentation/spec.yaml:8]: filter on ui is "image-segmentation" .. :)

Comment (yiyoulin) [reply assets/models/system/automl_instance_segmentation/spec.yaml:8]: discussed offline. no need to change.

**Review by sharma-riti — APPROVED** [review]
